concourse/scripts/build_ip4r.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash -l
+
+set -exo pipefail
+
+CWDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+TOP_DIR=${CWDIR}/../../../
+
+source "${TOP_DIR}/gpdb_src/concourse/scripts/common.bash"
+function pkg() {
+    [ -f /opt/gcc_env.sh ] && source /opt/gcc_env.sh
+    source /usr/local/greenplum-db-devel/greenplum_path.sh
+
+    if [ "${IP4R_OS}" = "rhel6" ]; then
+        export CC="$(which gcc)"
+    fi
+
+    pushd /home/gpadmin/ip4r_src
+    # local last_release_path
+    # last_release_path=$(readlink -e /home/gpadmin/last_released_ip4r_bin/ip4r-*.tar.gz)
+    make clean
+    make install
+    popd
+
+    pushd /usr/local/greenplum-db-devel/
+    echo 'cp -r lib share $GPHOME || exit 1'> install_gpdb_component
```

Comment:
let's create an independent script to do this? like `pack.sh`. So we can make a `tar.gz` on our dev machine as well.

Reply:
This script `pack.sh` will require two arguments? 1. greenplum installation path. 2. Generated package name/path.